packages/plugin-css/package.json

```diff
@@ -47,9 +38,9 @@
     "svgo": "^3.2.0"
   },
   "devDependencies": {
-    "@cobalt-ui/cli": "^1.6.2",
-    "@cobalt-ui/core": "^1.6.1",
-    "@types/node": "^20.11.6",
-    "vitest": "^1.2.1"
+    "@cobalt-ui/cli": "workspace:^",
```

Comment:
I’m giving up. pnpm wins. It’ll be `workspace:` now and everything will be fine.